app/src/main/java/amazingme/activities/app/MainMenu.java

```diff
@@ -31,6 +31,8 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        // Register games when app is launched
+        GameRegister.registerGames(getContext());
```

Comment:
i would rather just a call to EnumeratedActivities.getRegisteredGames()

i have it ready to go but don't want to cause you a merge conflict. 

if you just want to push, that's fine. I can change it locally.